packages/office-ui-fabric-react/src/components/Calendar/CalendarDay.tsx

```diff
@@ -161,6 +163,7 @@ export class CalendarDay extends BaseComponent<ICalendarDayProps, ICalendarDaySt
                 { strings.shortDays.map((val, index) =>
                   <th
                     className={ css('ms-DatePicker-weekday', styles.weekday) }
+                    role='tablist'
```

Comment:
I don't think these are really tabs, they're grid cells. :) can we put role='grid' here? Then we can put role='row' on the tr element, and role='gridcell' down where you changed the role from button to tab. that should make it a little more consistent.